modules/caddyhttp/reverseproxy/fastcgi/fastcgi.go

```diff
@@ -74,6 +76,11 @@ type Transport struct {
 	// The duration used to set a deadline when sending to the FastCGI server.
 	WriteTimeout caddy.Duration `json:"write_timeout,omitempty"`
 
+	// Capture and log any messages sent by the upstream on stderr. Logs at WARN
+	// level by default. If the response has a 4xx or 5xx status ERROR level will
+	// be used instead.
+	CaptureStderr bool `json:"capture_stderr,omitempty"`
```

Comment:
Wonder if they should be emitted at DEBUG level unless the response is 5xx (then WARN)? I imagine the _error itself_ is already emitted at an ERROR-level log?

Reply:
I'd say it makes sense to log at WARN if they turned on this option. They opted in to want to hear about errors, so we should make it "visible".